ui/pages/first-time-flow/onboarding-initiator-util.js

```diff
@@ -1,25 +1,18 @@
 import browser from 'webextension-polyfill';
 import log from 'loglevel';
 
-const returnToOnboardingInitiatorTab = async (onboardingInitiator) => {
-  const tab = await new Promise((resolve) => {
-    browser.tabs.update(
-      onboardingInitiator.tabId,
-      { active: true },
-      // eslint-disable-next-line no-shadow
-      (tab) => {
-        if (tab) {
-          resolve(tab);
-        } else {
-          // silence console message about unchecked error
-          if (browser.runtime.lastError) {
-            log.debug(browser.runtime.lastError);
-          }
-          resolve();
-        }
-      },
+export const returnToOnboardingInitiatorTab = async (onboardingInitiator) => {
+  let tab;
+  try {
+    tab = await browser.tabs.update(onboardingInitiator.id, {
+      active: true,
+      highlighted: true,
```

Comment:
Why was this option added?

Reply:
Added while I was testing and forgot to remove. It's not necessary. Will remove.

Reply:
updated here: https://github.com/MetaMask/metamask-extension/pull/14550/commits/2da95286b25184a93cfe9d9691913e48f91aa25d